addons/udes_stock/models/stock_inventory.py

```diff
@@ -0,0 +1,97 @@
+from odoo import models, fields
+
+
+class Inventory(models.Model):
+    _inherit = "stock.inventory"
+
+    # Make locations field mandatory
+    location_ids = fields.Many2many(required=True)
```

Comment:
Question:
Why are these changes thought to be here and not in udes_stock_check module?

In that case should we consider that overriding location_ids in udes_stock and removing at all in udes_stock_check.
Because we created udes_stock_check for stock inventory changes it sound better to me that these changes are in udes_stock_check

Reply:
So I want to be able to offer a better interface by default with UDES, so if we ever have a custom module later down the line, we could have "the-boy-who-lived" which doesn't want the "you-know-who" functionality.

But I take the point that `udes_stock` is becoming bloated, which we want to avoid. So will put these changes into `udes_stock_inventory`.

Reply:
Moved the functionality to a new module.